R/calculate.R

```diff
@@ -110,30 +110,36 @@
 #' # you can use calculate on greta arrays created even after the inference on
 #' # the model - e.g. to plot response curves
 #' petal_length_plot <- seq(min(iris$Petal.Length),
-#'                          max(iris$Petal.Length),
-#'                          length.out = 100)
+#'   max(iris$Petal.Length),
+#'   length.out = 100
+#' )
 #' mu_plot <- alpha + petal_length_plot * beta
 #' mu_plot_draws <- calculate(mu_plot, values = draws)
 #' mu_est <- colMeans(mu_plot_draws[[1]])
-#' plot(mu_est ~ petal_length_plot, type = "n",
-#'      ylim = range(mu_plot_draws[[1]]))
+#' plot(mu_est ~ petal_length_plot,
+#'   type = "n",
+#'   ylim = range(mu_plot_draws[[1]])
+#' )
 #' apply(mu_plot_draws[[1]], 1, lines,
-#'       x = petal_length_plot, col = grey(0.8))
+#'   x = petal_length_plot, col = grey(0.8)
+#' )
 #' lines(mu_est ~ petal_length_plot, lwd = 2)
 #'
 #' # trace_batch_size can be changed to trade off speed against memory usage
 #' # when calculating. These all produce the same result, but have increasing
 #' # memory requirements:
 #' mu_plot_draws_1 <- calculate(mu_plot,
-#'                              values = draws,
-#'                              trace_batch_size = 1)
+#'   values = draws,
+#'   trace_batch_size = 1
+#' )
 #' mu_plot_draws_10 <- calculate(mu_plot,
-#'                               values = draws,
-#'                               trace_batch_size = 10)
+#'   values = draws,
+#'   trace_batch_size = 10
+#' )
 #' mu_plot_draws_inf <- calculate(mu_plot,
```

Comment:
It's weird to me that there are false negatives like this (`mu_plot` should be on a new line)

Reply:
It's deliberate and according to the tidyverse style guide: https://style.tidyverse.org/syntax.html#long-lines: The line is not broken before the first argument if there are named arguments in the call.

Reply:
Yeah, I see that now - thanks!
I think (the other) Nick is right though that the behaviour in the lapply calls below is not quite consistent with the style; though the wording of the style guide here is kind of ambiguous.